zulipterminal/ui_tools/views.py

```diff
@@ -113,7 +113,8 @@ def load_old_messages(self, anchor: int) -> None:
         ids_to_keep = self.model.get_message_ids_in_current_narrow()
         if self.log:
             top_message_id = self.log[0].original_widget.message['id']
-            ids_to_keep.remove(top_message_id)  # update this id
+            if top_message_id in ids_to_keep:
+                ids_to_keep.remove(top_message_id)  # update this id
```

Comment:
Is this a separate bugfix?

Reply:
This is to fix a race round condition when inside the current starred narrow, we unstar and remove the first message from the top, but when loading older messages, we try to remove this message_id again which is technically not present in the set anymore throwing errors.

Reply:
This is more of a hypothesis, rather than an exact cause. I tested unstaring and moving the cursor up with and without the if block. Without the if block, I go an exception twice for trying to remove the message_id, which is not present. This although didn't occur with the if block. That's why added the if block.

Reply:
Update: I tested again without the if block, and found the error. 
Steps to check it out:
- Remove the if block.
- Move to the topmost loaded message inside the starred_messages narrow. Ensure that new messages don't get loaded.
- Unstar it.
- Try moving up(loading older messages).

Reply:
![Screenshot from 2021-03-08 17-52-13](https://user-images.githubusercontent.com/48902681/110320964-0f1c4680-8037-11eb-8193-0c47c8601e4a.png)